packages/jest-preset-mc-app/package.json

```diff
@@ -23,7 +23,6 @@
   "dependencies": {
     "@babel/core": "7.4.5",
     "@commercetools-frontend/babel-preset-mc-app": "13.9.0",
-    "@commercetools/enzyme-extensions": "3.0.1",
```

Comment:
Bye bye 👋

Reply:
Although it might be a breaking change now that I think about it...hmm we need to find a better solution I guess 🤔